.github/workflows/ms.aad.domainservices.yml

```diff
@@ -34,31 +35,37 @@ env:
   DEPLOYMENT_SP_ID: '${{ secrets.DEPLOYMENT_SP_ID }}'
 
 jobs:
-  ############################
-  #   SET INPUT PARAMETERS   #
-  ############################
-  job_set_workflow_param:
+  ###########################
```

Comment:
Please rename the GH workflow to 'AAD: DomainServices'

Reply:
@eriqua, done